MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,50 @@
+# Migration Guide
+
+## Upgrading from v2 -> v3
+
+### Improvements and changes
+
+- Web Auth will now have default scope of 'openid profile email', so these scopes can be removed if you're explicitly specifying them
+- Minimum supported version for iOS is bumped to 13
+- Revoke Token and Change Password now return `void` instead of an empty object
+
+### Breaking changes
+
+- The properties inside the `user` object will now be camelCase instead of snake_case
+- Removed `type` property returned in Android. Use `tokenType` instead.
+- `getCredentials` in Android will return `expiresIn` instead of `expiresAt`
+- `max_age` parameter is changed to `maxAge` in `WebAuth.authorize()`
+- `skipLegacyListener` has been removed in `authorize` and `clearSession`
+- `customScheme` is now part of `ClearSessionOptions` instead of `ClearSessionParameters` in `clearSession`
+
+### Callback URL migration
+
+We are migrating the callback URL we use for the SDK to below.
+
+**Old**
+
+```
+iOS: {PRODUCT_BUNDLE_IDENTIFIER}://{DOMAIN}/ios/{PRODUCT_BUNDLE_IDENTIFIER}/callback
+Android: {YOUR_APP_PACKAGE_NAME}://{DOMAIN}/android/{YOUR_APP_PACKAGE_NAME}/callback
```

Comment:
Are these examples of the old custom scheme URLs accurate? I thought it would be good to contrast the old URLs vs the new so that developers can see the difference.